jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -240,15 +267,47 @@ async def get(self, path):
         self._finish_model(model, location=False)
 
 
+# TODO: see about writing to/from config file
```

Comment:
Reading from a config file makes sense, but what do we gain by writing a config? If running this in a container, then writing to a config doesn't have any effect as the only time you'd reconnect would be if the container restarted in which case the file doesn't exist, right?

Reply:
i think another benefit of writing the config to file is that it enables pachctl to work after you log in via the extension. also if i'm not mistaken, the config file doesn't necessarily not persist between container restarts, right?